src/components/homeData.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="homeData">
+    <div v-bind:class="userInfo">
+      <p>{{currentUser.name}}さんようこそ！！</p>
+      <div v-bind:class="optionBox">
+        <p>残高：{{currentUser.balance}}</p>
+        <button v-bind:class="logoutBtn">ログアウト</button>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+  export default{
+    name: 'homeData',
+    data: function() {
+      return{
+        userInfo: 'user-info',
+        optionBox: 'option-box',
+        logoutBtn: 'logout-btn',
+      };
+    },
+    created: function() {
+      this.$store.dispatch('getUser');
+    },
+    methods: {
+     
+    },
+    computed: {
+      currentUser() {
+        const currentUser = this.$store.getters.user;
+        return currentUser;

```

Comment:
定数currentUserに格納する意味は特に無いかと思うので、gettersの値を直接returnしても良いかと思います。
```suggestion
        return this.$store.getters.user;

```